scripts/pc2sandbox.sh

```diff
@@ -189,8 +248,9 @@ TIMELIMIT_US=$((TIMELIMIT * 1000000))
 DEBUG echo setting cpu limit to $TIMELIMIT_US microseconds "("ulimit -t $TIMELIMIT ")"
 ulimit -t $TIMELIMIT
 
-DEBUG echo setting maximum user processes to $MAXPROCS + whatever the user is currently using
-ulimit -u $((MAXPROCS+`ps -T -u $USER | wc -l`))
+MAXPROCS=$((MAXPROCS+`ps -T -u $USER | wc -l`))
+DEBUG echo setting maximum user processes to $MAXPROCS
+ulimit -u $MAXPROCS
 
 # Remember wall time when we started
 starttime=`GetTimeInMicros`
```

Comment:
With this call being located here, the user is being charged for all of the time taken below to execute debugging output (if enabled), but also for the time to do stuff like set the `trap` handler.  It seems like this call should be made as closely as possible to the `setsid taskset`...

Reply:
Repeat